inappmessaging/USERGUIDE.md

```diff
@@ -323,6 +322,10 @@ Documents targeting Product Managers:
 
 ## <a name="changelog"></a> Changelog
 
+### 4.1.1-PP (in-progress)
+* SDKCF-3781: Fixed crash due to failing master key generation for encrypted shared preferences.
```

Comment:
Can you add inline links to the Google bugs please, like
The root cause of the crash is suspected to be these Android platform issues [#123456](https://google) and [#78910](https://google). To prevent the crash the following workaround is implemented:

Reply:
Updated based on suggestion